ReSharper.FSharp/src/FSharp/FSharp.Psi.Services/src/Util/TypeAnnotationsUtil.fs

```diff
@@ -6,7 +6,7 @@ open JetBrains.ReSharper.Psi.Tree
 let rec private visitPattern (acc: ITreeNode list) (pattern: IFSharpPattern) =
     match pattern with
     | null
-    | :? IUnitPat
+    | :? IConstPat
```

Comment:
Is this change covered in any test?

Reply:
https://github.com/JetBrains/resharper-fsharp/blob/5f7806afa60ca104aa2c6070d25ddd88e8e61aa1/ReSharper.FSharp/test/data/features/daemon/typeHints/_TestSource.fs#L10

But it also makes sense to add some literal to parameters in these tests also